index.html

```diff
@@ -0,0 +1,362 @@
+    </article>
+  </div>
+  <div class="column is-offset-1 is-2">
+    
+  <article class="media">
+    <figure class="media-left pb-5">
+      <p class="image is-128x128">
+        <img src="images/speaker1.png">
+      </p>
+    </figure>
+    
+      <div class="content">
+        <p>
+          <strong>John Smith</strong> 
+          <br>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin ornare magna eros,  quis feugiat facilisis.
+        </p>
+      </div>
+      
+   
+    
+  </article>
+</div>
+</div>
+<div class="columns is-centered">
+  <div class="column is-offset-1 is-2">
+    
+  <article class="media">
+    <figure class="media-left pb-5">
+      <p class="image is-128x128">
+        <img src="images/speaker1.png">
+      </p>
+    </figure>
+    
+      <div class="content">
+        <p>
+          <strong>John Smith</strong> 
+          <br>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin ornare magna eros, quis feugiat facilisis.
+        </p>
+      </div>
+      
+   
+    
+  </article>
+</div>
+<div class="column is-offset-1 is-2" >
+  
+<article class="media">
+  <figure class="media-left pb-5">
+    <p class="image is-128x128">
+      <img src="images/speaker1.png">
+    </p>
+  </figure>
+  
+    <div class="content">
+      <p>
+        <strong>John Smith</strong> 
+        <br>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin ornare magna eros, quis feugiat facilisis.
+      </p>
+    </div>  
+</article>
+</div>
+</div>
+<div class="partner mt3 pt-3 mb-0 ">
+  <div class="columns is-centered">
+    <div class="column has-text-centered">
+      <h4
+        class="title is-4 has-text-grey mt-2 mb-5 has-text-centered">
+        Partners
+      </h4>
+     </div>
+    
+  </div>
+  <div class="columns is-centered mt-0 mb-0 ">
+    <div class="column is-2 pt-5 mt-5 mb-0">
+      <figure class="image is-128x128">
+        <img src="https://www.freepnglogos.com/uploads/google-logo-png/file-google-logo-svg-wikimedia-commons-23.png"alt="google logo" />
+      </figure>
+     
+    </div>
+    <div class="column is-2 pt-5 mt-5 mb-0">
+      <figure class="image is-128x128">
+        <img src="https://www.freepnglogos.com/uploads/netflix-tv-logo-png-9.png" alt="netflix tv logo png" />
+      </figure>
+     
+    </div>
+    <div class="column is-2 pt-5 mt-5 mb-0">
+      <figure class="image is-128x128">
+        <img src="https://www.freepnglogos.com/uploads/twitter-logo-png/twitter-logo-with-birds-symbol-icon-24.png"alt="twitter logo with birds symbol icon" />
+      </figure>
+     
+    </div>
+    <div class="column is-1 pt-5 mt-5 mb-0">
+      <figure class="image is-128x128">
+        <img src="https://www.freepnglogos.com/uploads/spotify-logo-png/spotify-attempts-clarify-lack-google-cast-support-13.png" width="200" alt="spotify attempts clarify lack google cast support" />
+      </figure>
+     
+    </div>
+    
+  </div>
+    
+  </body>
```

Comment:
- [ ] On the mobile view of the Home page, this partner section should not be visible as you have done here. 
- [ ] On the desktop view, the partner icons are arranged in a specific manner. Kindly fix that
- [ ] Please note that on the desktop view of both the Home page and About page, there should be a footer section but you have not added it. 

- [ ] ❗ Please be aware that the background color of the footer on the Home page is **_white_**, and it includes a designated Partner section.

Original design - Home page Desktop
<img width="467" alt="image" src="https://github.com/Drew246/capstone-project-1/assets/84156714/1a0628c6-95a9-4dbe-9eca-73c5c48baae0">


- [ ] ❗_**Please take note that the footer on the about page is set to a black background color, and notably, there is an absence of a Partner section in this context. In other words, the footer has a black background, and there is no section dedicated to Partners on the about page.**_

Original design About page
<img width="417" alt="image" src="https://github.com/Drew246/capstone-project-1/assets/84156714/20622c30-72cb-4c62-99ca-028cb16e4a2d">